slices/debconf.yaml

```diff
@@ -0,0 +1,24 @@
+package: debconf
+
+essential:
+  - debconf_copyright
+
+slices:
+  bins:
+    contents:
+      /usr/bin/debconf:
+      /usr/bin/debconf-apt-progress:
+      /usr/bin/debconf-communicate:
+      /usr/bin/debconf-copydb:
+      /usr/bin/debconf-escape:
+      /usr/bin/debconf-set-selections:
+      /usr/bin/debconf-show:
+      /usr/sbin/dpkg-preconfigure:
+      /usr/sbin/dpkg-reconfigure:
```

Comment:
I think this package slice is still missing the following:
- perl module files (everything under `/usr/share/perl5`)
- bash autocompletion scripts (under `/usr/share/bash-completion/completions`)
- some other bash/perl scripts (under `/usr/share/debconf`)

Refer to https://packages.ubuntu.com/noble/all/debconf/filelist for the full list of files.

Reply:
All under the `bins` slice? 

If not, which names do you suggest?

Reply:
I think `bins` would be good enough.

Reply:
it should be in separate slice(s) - see https://github.com/canonical/chisel-releases/pull/224#pullrequestreview-2070380422